SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -144,14 +144,7 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 
 	if (Class->HasAnySpatialClassFlags(SPATIALCLASS_Singleton))
 	{
-		if (Class->HasAnySpatialClassFlags(SPATIALCLASS_ServerOnly))
-		{
-			ComponentDatas.Add(improbable::ServerOnlySingleton().CreateServerOnlySingletonData());
```

Comment:
We should remove the schema component as well as the `ServerOnlySingleton` class.